doc/architecture/diagrams/data-structures/refer.wsd

```diff
@@ -0,0 +1,66 @@
+@startuml
+skinparam linetype ortho
+
+package "Things we own" {
+  entity Referral {
+    e.g. The referral of a person in prison or on probation to a Course
+    --
+    * id: uuid
+    * prisonNumber: text
+    * offering: CourseOffering
+    reason: text
+    referrerId: text
+    oasysConfirmed: boolean
+    reason: text
+    submitted: boolean
+  }
+
+
+  entity HistoricCourse {
+    e.g. Previous course that the person has been on, that aren't in our own service
+    Will have either a courseId or otherCourseName
+    --
+    * id: uuid
+    * prisonNumber: text
+    courseId?: uuid
+    otherCourseName?: text
+    yearStarted?: text
```

Comment:
I think yearStarted should be represented by an optional integer.  It will then be possible to search by year ranges, which wouldn't be possible with a string.